src/Directory.Build.props

```diff
@@ -53,6 +53,20 @@
 		<ExtrasEnableDefaultPageItems>false</ExtrasEnableDefaultPageItems>
 		<EnableDefaultPageItems>false</EnableDefaultPageItems>
 		<EnableAutomaticXamlPageInclusion>true</EnableAutomaticXamlPageInclusion>
+
+		<NetLatest>net8.0</NetLatest>
+		<NetPrevious>net7.0</NetPrevious>
+
+		<NetLatestNetCoreMobile>$(NetLatest)-android;$(NetLatest)-ios;$(NetLatest)-maccatalyst;$(NetLatest)-macos</NetLatestNetCoreMobile>
+		<NetPreviousNetCoreMobile>$(NetPrevious)-android;$(NetPrevious)-ios;$(NetPrevious)-maccatalyst;$(NetPrevious)-macos</NetPreviousNetCoreMobile>
+
+		<NetWasmPreviousAndLatest>net7.0;net8.0</NetWasmPreviousAndLatest>
```

Comment:
When we switch Wasm from `netX.0` to `netX.0-browser`, we will (hopefully) need to only update this line (and the `nuspec`s)